accounts/migrations/0016_myuser_nickname.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations, models
+from accounts.utils.generate_default_name import generate_default_name
+
+
+def forwards_func(apps, schema_editor):
+    MyUser = apps.get_model("accounts", "myuser")
+    my_users = MyUser.objects.select_for_update()
+    for my_user in my_users:
+        my_user.nickname = generate_default_name()
```

Comment:
they need to be unique, and that should be enforced in this case (i.e. keep generating until it doesn't clash with an existing nickname in the course)